lib/core/include/transport/default_transport.hpp

```diff
@@ -267,6 +267,10 @@ namespace irods::experimental::io::NAMESPACE_IMPL
 
             _func(input);
 
+            if (input.openFlags | O_CREAT) {
```

Comment:
This looks like it should be `if ((input.openFlags & O_CREAT) == O_CREAT)`.

Reply:
Besides your being correct, this is a residual diff from an experiment during development of this change. Will delete. Thanks!